docs/components/board/customlinux.md

```diff
@@ -9,11 +9,27 @@ tags: ["board", "components"]
 # SMEs: Susmita
 ---
 
-Configure a `customlinux` board to integrate linux boards like the [Pumpkin i500](https://ologicinc.com/portfolio/mediateki500/) into your robot.
+The `customlinux` board model supports boards like the [Pumpkin i500](https://ologicinc.com/portfolio/mediateki500/) that run Linux operating systems and are not supported by other built-in Viam models.
+
+To integrate a custom Linux board into your robot:
```

Comment:
Just noting that this is the only instance of `customlinux` not all one word and monospace, in case this wasn't intended. I can see reasons for either way though, so no action unless not intended!

Reply:
Indeed--it was intentional because I like to use real words on occasion :) but if folks think it should be monospace I'm fine with changing it!

Reply:
Not at all -- makes sense to me!